aips/objects.md

```diff
@@ -43,22 +42,23 @@ The existing Aptos data model emphasizes the use of the `store` ability within M
 - Events cannot be emitted from data but from an account that may not be associated with the data.
 - The existing data model makes it challenging to provide rich accessibility models as behavior must be defined well before applications can be built. This leads to excessively complex security policies and unnecessary data to support flexible configuration.
 - The existing Move model has only a single capability — signer, everything else must be defined and codified by the developer of a module.
+- Transferring logic is limited to the APIs provided in the respective modules and generally requires loading resources on both the sender and receiver adding unnecessary cost overheads.
 
 ## Specification
 
 ### Overview
 
 Objects are built with the following considerations:
 
-- Allow for storing ownership of heterogeneous types: `OwnerRef`.
+- Simplified storage interface for heterogeneous types.
 - Enable a rich and dynamic data model while limited impact on storage, i.e., storage should not take up more than effectively a single resource (storage slot): see Resource Groups AIP.
 - Data is immobile but ownership still remains portable: storing data in resources and ownership within `OwnerRef`.
 - Make easily usable from entry functions: each object is stored at an address, which is a supported API type.
 - Offer rich set of capabilities: the `Ref` model.
 - Support emitting events directly: directly built-in.
 - Simplified access to storage avoiding costly deserialization, serialization: resources, once loaded, have limited costs associated with future access.
-- Objects are fully deletable.
-- Objects must be fully instantiated at creation time and cannot have new resources added afterward
+- Objects are deletable.
```

Comment:
is this setting a flag, or dropping everything inside the group and setting a flag?